### DISCUSSION TITLE
"./run_tests error"

### DISCUSSION THREAD

**Discussion by snugook1108** [idaholab/moose#19610] [Q&A General] [answered]
When I run the "./run_test", I received the error message like this. 
```
our reference and displaced meshes are not in sync with respect to whether we should delete remote elements.
```

How to handle this problem after two applications coupled with installation?

**Comment by GiudGiud** [comment, marked as answer]
Hello

Is this for MOOSE? What applications are included?

Guillaume

**Reply by snugook1108** [reply]
Thank you for the reply, It is the MOLE and SAM coupling.

**Reply by GiudGiud** [reply]
ok seems it s related to https://github.com/idaholab/moose/discussions/18736

Tagging @rtaylo45 

I think the test that triggers this has been modified?

**Reply by snugook1108** [reply]
Before the coupling, the tests of Mole are working ok, but After the coupling, the tests of Mole are not working

**Reply by GiudGiud** [reply]
Unfortunately I dont have access to Mole so I wont be much help here. 

SAM uses the displaced mesh extensively, so they may add by default some custom treatment for it, which you may be missing if the coupled application is not a SAMApp. @travismui do you know what is added in SAM?

**Reply by rtaylo45** [reply]
this is related to using periodic boundary conditions with SAM displaced mesh. Basically, you can't use periodic BCs as of now. I did not look into why this was happening exactly. @snugook1108

**Reply by rtaylo45** [reply]
You need to run the test as `./run_tests -g sam_coupled`. This will skip any mole test that includes the periodic boundary condition. The `-g` is for group. Its the current way i use to tell mole which specific test to run when SAM is coupled. In the future it would be nice for Mole to only run test based on makefile variables.
